cmd/celestia-appd/cmd/init.go

```diff
@@ -0,0 +1,21 @@
+package cmd
+
+import (
+	"github.com/cosmos/cosmos-sdk/x/genutil/client/cli"
+	"github.com/spf13/cobra"
+
+	"github.com/celestiaorg/celestia-app/v3/app"
+)
+
+// InitCmd returns a custom version of the default init command with improved help docs.
+func InitCmd() *cobra.Command {
+	cmd := cli.InitCmd(app.ModuleBasics, app.DefaultNodeHome)
+
+	cmd.Short = "Initialize configuration files for a Celestia consensus node"
+	cmd.Long = `
+This command sets up a genesis file and default configuration for either a validator or a full node.`
+
+	cmd.Example = "celestia-appd init testnode --chain-id testnet-1"
```

Comment:
testnet-1 isn't a valid testnet name. Let's copy the command from https://docs.celestia.org/how-to-guides/consensus-node#set-up-the-p2p-networks

```suggestion
	cmd.Example = "celestia-appd init node-name --chain-id celestia"
```